hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -256,6 +256,86 @@ def hpat_pandas_series_values_impl(self):
     return hpat_pandas_series_values_impl
 
 
+@overload_method(SeriesType, 'var')
+def hpat_pandas_series_var(self, axis=None, skipna=None, level=None, ddof=1, numeric_only=None):
```

Comment:
skipna=True?

Reply:
By default `skipna=None` in Pandas interface.